src/cljdoc/render/offline.clj

```diff
@@ -171,9 +173,9 @@
         lib-page-features (->> doc-attrs (map :page-features) (apply merge))]
     (reduce
      into
-     [[["assets/cljdoc.css" (io/file (io/resource "public/cljdoc.css"))]]
-      [["assets/js/index.js" (io/file "resources-compiled/public/js/index.js")]]
-      [["assets/js/index.js.map" (io/file "resources-compiled/public/js/index.js.map")]]
+     [[["assets/cljdoc.css" (io/file (str static-resource-prefix (get static-resources "/main.css")))]]
+      [["assets/js/index.js" (io/file (str static-resource-prefix (get static-resources "/main.js")))]]
+      [["assets/js/index.js.map" (io/file (str static-resource-prefix (get static-resources "/main.js" ".map")))]]
```

Comment:
@rvalentini Here you removed `io/resource` and instead provided the resource directory as a process-relative path. I think we should continue to use `io/resource`, e.g.

```clj
(io/resource (str "public/out" (get static-resources "/main.css")))
````

(or something like that :) 

Duh, now that I write this I notice this is only true for the first asset. Overall I feel like it's preferable to serve from classpath as that is packaged with Jars etc. Maybe we can use `io/resource` for all these resources here?